### PR TITLE
Update the tag of the docker image in the targeted repos

### PR DESCRIPTION
**Pull request by hectorj2f**
related to: https://github.com/fabric8-ui/fabric8-ui-openshift-nginx/issues/14

### CODE REVIEW

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-1`

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-14`

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-16`

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-17`

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+@Library('github.com/hectorj2f/fabric8-pipeline-library@dockerbaseimage9')
```
Comment (hectorj2f) [inline Jenkinsfile]: this needs to be changed to the upstream repo once this is merged into the pipeline library

File: Jenkinsfile [hunk 2]
```diff
@@ -50,5 +51,21 @@ dockerNode{
...
+        updateDownstreamRepoDependencies(version)
       }
 }
+}
+
+def updateDownstreamRepoDependencies(v) {
+  pushNewDockerImageTagChangePR {
+    propertyName = 'fabric8\\/fabric8-openshift-nginx'
+    projects = [
+            'fabric8-ui/fabric8-ui',
```
Comment (hectorj2f) [inline Jenkinsfile]: I added the two projects that require an update of the base image tag.

**Review by rawlingsj — APPROVED** [review]
LGTM

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-19`

**Comment by fabric8cd** [comment]
@hectorj2f snapshot fabric8-ui-openshift-nginx image is available for testing.  `docker pull fabric8/fabric8-openshift-nginx:SNAPSHOT-PR-15-23`

**Review by aslakknutsen — APPROVED** [review]

**Pull request merged**